### ISSUE TITLE
When trying to download a module I get DialogHelper is deprecated error

### ISSUE THREAD

**Issue by sgtsaughter** [hechoendrupal/DrupalConsole#1086]
When I try to run the command "drupal module:download [module name]" I receive the following error:

Symfony\Component\Console\Helper\DialogHelper" is deprecated since version 2.5 and will be removed in 3.0. Use "Symfony\Component\Console\Helper\QuestionHelper" instead

This prevents me from downloading the module.

**Comment by jmolivas** [comment]
Is this an exception not a warning?

Should be a warning message not a exception, let me know if you can help me test a temporary fix. Until we replace `DialogHelper` with `QuestionHelper`.

**Comment by sgtsaughter** [comment]
I'm not too familiar with Symphony or the code for the console module, but I am available to help test a fix if you let me know what needs to be done.

**Comment by jmolivas** [comment]
Do you have git?
Did you fork and clone the project?
May using the support channel is easy for this communication: https://gitter.im/hechoendrupal/DrupalConsole

**Comment by bojanz** [comment]
Seeing this on several machines today (we're doing a mini-training). Seems to stop the process, nothing gets downloaded.

**Comment by bojanz** [comment]
I've created a command that uses the [new helper](http://symfony.com/doc/current/components/console/helpers/questionhelper.html), it works fine, but the output is black&white, no colors:
https://github.com/commerceguys/commerce/commit/30cc62ea6ca7c71a5237af91bdaa578d47010042

**Comment by jmolivas** [comment]
@sgtsaughter: Deprecated messages got removed and fixed with on v0.9.9 release.

**Comment by jmolivas** [comment]
@bojanz: We are using http://symfony.com/blog/new-in-symfony-2-8-console-style-guide 

#### You can take advantage of those helper methods by using:
```
protected function interact(InputInterface $input, OutputInterface $output)
{

  $io = new DrupalStyle($input, $output);
  ...
}
```
![io-ask](https://cloud.githubusercontent.com/assets/366275/11832144/29e6832c-a36a-11e5-8704-daeb7a2d0e29.png)
![io-choice](https://cloud.githubusercontent.com/assets/366275/11832143/29e630ac-a36a-11e5-96a2-cbb46af75a76.png)
![io-confirm](https://cloud.githubusercontent.com/assets/366275/11832145/29eb8d40-a36a-11e5-814f-3bbd14abc392.png)

#### We add `DrupalStyle` to: 
- Accept empty values on ask, hidden and choice.
- Avoid listing choice values on questions.  

> NOTE: We are planning to extract `DrupalStyle' class and the other added classes to a separated project and use as dependency.

**Comment by darol100** [comment]
Closing this issue since....

> Deprecated messages got removed and fixed with on v0.9.9 release.
